x/documents/client/rest/query.go

```diff
@@ -51,6 +51,15 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
 // --- Documents
 // ----------------------------------
 
+// @Summary Get the sent documents
+// @Description This endpoint returns the sent documents, along with the height at which the resource was queried at
+// @ID getSentDocumentsHandler
+// @Produce json
+// @Param address path string true "Address of the user"
+// @Success 200 {object} x.JSONResult{result=[]types.Document}
+// @Failure 404
+// @Router /documents/{address}/sent [get]
```

Comment:
Rest endpoint is `docs` not `documents`. 
For example: if you call

http://localhost:1317/documents/did:com:12p24st9asf394jv04e8sxrl9c384jjqwejv0gf/sent

return error `404`